src/models/tasks.py

```diff
@@ -0,0 +1,78 @@
+from transformers import BertConfig, BertForSequenceClassification
+import torch
+from torch import nn
+from loguru import logger
+import numpy as np
+from torch.nn.utils.rnn import pad_sequence
+from torch.nn import CTCLoss
+
+from src.models.vtr.ocr import OCRHead
+from src.utils.common import PositionalEncoding
+from src.models.embedders.vtr import VTREmbedder
+from src.utils.common import char2int
+
+
+def compute_ctc_loss(
```

Comment:
I'd move this function to the `utils` directory. common.py or create new file losses.py